extensions/blocks/donations/tab.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * WordPress dependencies
+ */
+import { __experimentalBlock as Block, RichText } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Context from './context';
+import formatCurrency, { CURRENCIES } from '@automattic/format-currency';
+import { minimumTransactionAmountForCurrency } from '../recurring-payments';
```

Comment:
If we're re-using this across more than one block, I think it would make sense to move it to `extensions/shared`.